Sources/Collections/Observable2DArray.swift

```diff
@@ -359,6 +388,14 @@ extension MutableObservable2DArray {
     sections[index] = section
     subject.next(Observable2DArrayEvent(change: .updateSections([index]), source: self))
   }
+  
+  /// Replace the entier 2d array with a new one forcing a reload
+  public func replace2D(with list: Observable2DArray<SectionMetadata, Item>)  {
```

Comment:
I think we should rename this to just `replace`. Replacing it with another `Observable2DArray` implies that we are doing 2d replace. 

We should also rename `list` to `array` because it's array.